apis/s3/src/test/java/org/jclouds/s3/blobstore/integration/S3BlobIntegrationLiveTest.java

```diff
@@ -74,4 +74,9 @@ public void testPutBlobTierArchiveMultipart() throws Exception {
          throw new SkipException("S3 does not allow setting Glacier storage class on putBlob", hre);
       }
    }
+
+   @Override
+   public void testPutHugeObject() throws Exception {
```

Comment:
Does this method have a purpose?  Subclasses will inherit this functionality without overriding it.